.cirrus.yml

```diff
@@ -1,5 +1,19 @@
 gcp_credentials: ENCRYPTED[1816835da1e936dabb469b51501856ec8468676e35e967bd0fd720a815498e5ee6c8a6a79219ce273f67bcb8f1aa948a]
 
+tool_setup_template: &TOOL_SETUP_TEMPLATE
+  tool_setup_script:
+    - git fetch origin master # To set FETCH_HEAD for "git merge-base" to work
+    # Pinned version of the plugin tools, to avoid breakage in this repository
+    # when pushing updates from flutter/plugins.
+    - pub global activate flutter_plugin_tools 0.0.45+3
+
+flutter_upgrade_template: &FLUTTER_UPGRADE_TEMPLATE
+  upgrade_flutter_script:
+    - flutter channel $CHANNEL
```

Comment:
$CHANNEL seems not set.

Reply:
Oops, missed a piece. Fixed.

That also made me realize that we're currently only building the packages on master which seems... not great. So I've added stable, like we do in flutter/plugins.